### PR TITLE
132 1 update react peer devdependencies in package json react ions

### PR DESCRIPTION
**Pull request by nicoevergara**

### CODE REVIEW

**Comment by nicoevergara** [comment]
[(1) Update react peer/devDependencies in package.json (react-ions)](https://trello.com/c/tOxHQjV3/132-1-update-react-peer-devdependencies-in-package-json-react-ions)

File: package.json [hunk 1]
```diff
@@ -53,7 +53,7 @@
     "react-color": "github:getambassador/react-color#1.3.1",
     "react-dnd": "~2.1.4",
     "react-dnd-html5-backend": "~2.1.2",
-    "react-dom": "15.1.0",
+    "react-dom": "~15.5.4",
```
Comment (jivinivan) [inline package.json]: Let's remove the `react-dom` from `dependencies` and add to `peerDependencies` (and keep in `devDependencies`).

The benefit of only have `react` and `react-dom` in devDependencies/peerDependencies is the project can define the version it wants and there won't be multiple version of react/react-dom floating around your projects. 

Check out material-ui for reference https://github.com/callemall/material-ui/blob/master/package.json#L45

The only difference between us and them is we'll want to support a peerDependency range for react and react-dom (like you already have for react).

File: package.json [hunk 2]
```diff
@@ -133,6 +135,6 @@
     "webpack-hot-middleware": "~2.2.0"
   },
   "peerDependencies": {
-    "react": "~15.1.0"
+    "react": ">= 15.1.0 < 15.5.4"
```
Comment (jivinivan) [inline package.json]: This currently does not include `15.5.4`. We'll probably need to adjust to something like: `>= 15.1.0 < 15.6`

File: package.json [hunk 3]
```diff
@@ -113,12 +113,14 @@
     "postcss-load-config": "1.1.0",
     "postcss-loader": "1.2.2",
     "raw-loader": "~0.5.1",
-    "react": "15.1.0",
-    "react-addons-test-utils": "15.1.0",
+    "react": "15.5.4",
+    "react-addons-test-utils": "^15.5.1",
```
Comment (jivinivan) [inline package.json]: Could we remove this dependency now that `TestUtils` is part of `react-dom` or is it still needed?

**Pull request merged**